### DISCUSSION TITLE
If SSH keys can be used, can GPG keys be used, too?

### DISCUSSION THREAD

**Discussion by Hades32** [FiloSottile/age#280] [Q&A]
I'm asking especially because of your mention of GitHub on https://github.com/FiloSottile/age/blame/master/README.md#L112 which is pointing to the SSH keys, but there is also https://github.com/Hades32.gpg where you can get GPG keys

**Comment by FiloSottile** [comment]
We are not going to support them the same way SSH keys are supported, no. The OpenPGP key format is pretty awful, and the amount of users with a PGP key is a small fraction of those with an SSH key, so it is also much less useful as a convenience.
